providers/amazon/src/airflow/providers/amazon/aws/hooks/ses.py

```diff
@@ -101,6 +109,64 @@ def send_email(
             custom_headers=custom_headers,
         )
 
-        return ses_client.send_raw_email(
+        return self.conn.send_raw_email(
             Source=mail_from, Destinations=recipients, RawMessage={"Data": message.as_string()}
         )
+
+    async def asend_email(
```

Comment:
This comment applies to your other PR as well. Is prefixing the function with `a` a convention? I am just wondering if this is obivous to users/developers. Another alternative: prefixing with `async_`? I am not saying your way is wrong, I am unsure this would make sense for the majority of developers/users.

Reply:
Yeah, that was a discussion in a previous PR and there doesn't seem to be a real concrete consensus, both ways are used.  send/asend and get/aget are common enough to say they are standard for example, but (in the other PR, for example) publish/apublish is not really common enough to know if it's "standard".

Reply:
I guess it depends on personal preference :) Absolutely not blocking 👍